src/routes/+layout.svelte

```diff
@@ -43,6 +45,10 @@
 <Header />
 
 <main id="main-content" role="main">
+  {#if $userInfo}
```

Comment:
```suggestion
  {#if $userInfo.mainActivity}
```
Dommage d'instancier la modale pour rien... Ça ne marche pas comme ça?

Reply:
Ça marche aussi :)
Toutefois, je vais garder cette condition aussi dans la modale pour qu'elle porte aussi cette logique :)

Reply:
si tu le mets là, ça ne sert à rien de le remettre dans la modale a priori, elle va disparaitre du dom dès que la condition ne sera plus remplie.